dist/less/order-service.less

```diff
@@ -170,17 +175,24 @@
       }
     }
   }
+  .review-message {
+    padding-left: 10px;
+  }
+  .review-status {
+    display: flex;
+    font-size: 16px;
+
+    .pficon {
+      margin-top: 3px;
+    }
+  }
 }
 
-.wizard-pf-contents {
+.order-service-wizard-step.wizard-pf-main {
   @media (min-width: @screen-sm-min) {
-    display: flex;
-    min-height: 100%;
-    padding-bottom: (@grid-gutter-width / 2);
-    padding-top: (@grid-gutter-width / 2);
-    .scroll-shadows-horizontal-covers();
-    _::-webkit-full-page-media, _:future, :root & { // only target Safari
-      background: none;
-    }
+    height: calc(@order-service-page-height + @grid-gutter-width);
```

Comment:
> Based on my understanding of users and use cases, and applying the 80/20 rule, I would suggest that the behavior of the wizard be that the page size is consistent, and we use scrolling as needed.

Most of our templates have dozens of parameters. If we're applying the 80/20 rule, the common case is to have a lot, at least for the template broker. It's hard to say what other brokers will do.

> Whether that means we need to increase the minimum height to make sure that our "typical" number of parameters is shown without a scroll is a different question.

Even our full screen template page has a scrollbar. I don't think there is a minimum height that avoids it.

As someone who uses this dialog every day: having it resize doesn't bother me, but not being able to use the screen space makes it hard to use (see #177). I can only speak for myself, though.